src/sagemaker/workflow/step_collections.py

```diff
@@ -336,6 +349,8 @@ def __init__(
                 security_group_ids=estimator.security_group_ids,
                 description=description,
                 display_name=display_name,
+                repack_output_path=repack_output_path,
```

Comment:
Can you add a `repack_image_uri` argument? My customer is facing an issue where the python version `SKLearn` (3.7) estimator is different from the training image uri. That caused dependency errors when trying to install packages that are available on (for example) Python 3.8. Another option would be equating it to `estimator.training_image_uri()`.
```suggestion
                repack_output_path=repack_output_path,
                image_uri=estimator.training_image_uri(),
```